fbw-a32nx/src/systems/instruments/src/MsfsAvionicsCommon/common.scss

```diff
@@ -17,7 +17,36 @@
   font-size: 24px;
   fill: $display-green;
   visibility: visible !important;
-  
+
   text-anchor: middle;
   font-family: "Ecam", monospace;
 }
+
+.MaintenanceMode {
+  position: absolute;
+  left: 0%;
+  top: 0%;
+  width: 100%;
+  height: 100%;
+  border: none;
+}
+
+.EngineeringTestMode {
+  position: absolute;
+  left: 0%;
+  top: 0%;
+  width: 100%;
+  height: 100%;
+  border: none;
+  z-index: 999;
+  visibility: visible !important;
+  background-image: url('./thalesTest.svg');
```

Comment:
This image should be placed in the  `/Images/fbw-a32nx/` VFS folder. To achieve this, you can place it appropriately in `fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui`, and use the first path I mentioned in this style.

Reply:
Thanks for the pointer! I'm fairly new to this project. Fixed in latest commit.